ChustaSoft.GamersPlatformUtils.UI/Modules/Linker/LinkerControlModel.cs

```diff
@@ -0,0 +1,64 @@
+﻿using ChustaSoft.Common.Base;
+using ChustaSoft.Common.Models;
+using ChustaSoft.GamersPlatformUtils.Abstractions;
+using System.Collections.ObjectModel;
+using System.IO;
+
+namespace ChustaSoft.GamersPlatformUtils.UI.Modules.Linker
+{
+    public class LinkerControlModel : ViewModelBase
+    {
+
+        private bool _isLoading;
```

Comment:
IsLoading property is a bindable property highly related to visibility. The ViewModel is the site for this, the Model is more related to pure data, not to properties related to the actions of a control. 

I recommend moving this property to the ViewModel (LinkerControlViewModel)

Reply:
Property moved